exp-player/addon/components/exp-player.js

```diff
@@ -91,7 +91,16 @@ able to continue the study.
     currentFrameConfig: Ember.computed('frames', 'frameIndex', function() {
         var frames = this.get('frames') || [];
         var frameIndex = this.get('frameIndex');
-        return frames[frameIndex];
+        var config = frames[frameIndex];
```

Comment:
Rather than bundling the frame data into the configuration, why not pass a second variable into the component? It would also help to minimize side effects- if possible, we'd like to avoid changing the base behavior of frames that are part of other experiments (like Lookit) that share a codebase.